app/controllers/health_controller.rb

```diff
@@ -0,0 +1,13 @@
+class HealthController < ApplicationController
+  def workers
+    status = worker_health_checker.all_healthy? ? :ok : :internal_error
+
+    render json: worker_health_checker.statuses, status: status
```

Comment:
@jessieay I agree, if we add health checks for other things (ex the database) would you put them on separate controllers or this one?